### DISCUSSION TITLE
Confused about reconciliation

### DISCUSSION THREAD

**Discussion by comminutus** [fluxcd/flux2#4152] [General] [answered]
Hello, I created a Helm repository with `flux create source helm ...` and I can see the source on my cluster with `flux get source helm <repository name`.  I thought this would automatically create a git commit via automatic reconciliation but there are no new commits in my git repository.

How can I get flux to synchronize with my git repository so that my cluster is not in a state that my git repository doesn't know about?

**Comment by hiddeco** [comment, marked as answer]
Flux does not automatically sync to your Git repository, it automatically syncs _from_ your Git repository. You still need to commit the object you created to a Git repository which is synced using a `GitRepository` object, using for example `flux create source helm ... --export`.

**Reply by comminutus** [reply]
Ok, thanks from a flux newbie :)
